.travis.yml

```diff
@@ -41,38 +47,62 @@ jobs:
-        - openssl aes-256-cbc -K $encrypted_key -iv $encrypted_iv -in .ssh-keys/pub.enc -out ega-charts/localega/config/key.1.pub -d -md md5
-        - openssl aes-256-cbc -K $encrypted_key -iv $encrypted_iv -in .ssh-keys/ssl.enc -out ega-charts/localega/config/ssl.key -d -md md5
-        - openssl aes-256-cbc -K $encrypted_key -iv $encrypted_iv -in .ssh-keys/cert.enc -out ega-charts/localega/config/ssl.cert -d -md md5
-        - echo "JSUgLSotIG1vZGU6IGVybGFuZyAtKi0KJSUKW3tyYWJiaXQsW3tsb29wYmFja191c2VycywgWyBdIH0sIHt0Y3BfbGlzdGVuZXJzLCBbIDU2NzIgXSB9LCB7c3NsX2xpc3RlbmVycywgWyBdIH0sCiAgICAgICAge2hpcGVfY29tcGlsZSwgZmFsc2UgfSwge2RlZmF1bHRfdmhvc3QsICIvIn0sIHtkZWZhdWx0X3VzZXIsICAiZ3Vlc3QifSwKICAgICAgICB7ZGVmYXVsdF9wYXNzLCAgImd1ZXN0In0sIHtkZWZhdWx0X3Blcm1pc3Npb25zLCBbIi4qIiwgIi4qIiwiLioiXX0sCiAgICAgICAge2RlZmF1bHRfdXNlcl90YWdzLCBbYWRtaW5pc3RyYXRvcl19LCB7ZGlza19mcmVlX2xpbWl0LCAiMUdCIn1dfSwKICAgICAgICB7cmFiYml0bXFfbWFuYWdlbWVudCwgWyB7IGxpc3RlbmVyLCBbIHsgcG9ydCwgMTU2NzIgfSwgeyBzc2wsIGZhbHNlIH1dIH0sCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIHsgbG9hZF9kZWZpbml0aW9ucywgIi9ldGMvcmFiYml0bXEvZGVmcy5qc29uIn0gXX0KXS4=" | base64 -d > ega-charts/localega/config/rabbitmq.config
-        - echo "eyJyYWJiaXRfdmVyc2lvbiI6IjMuNi4xNiIsCiAgICAgInVzZXJzIjpbeyJuYW1lIjoiZ3Vlc3QiLCJwYXNzd29yZF9oYXNoIjoiSnpUdUp2SzA3VmlNMkhpODlETFMyTEw2dUVWVGNIdGtVQmszV0VDR2w2YmxpSERwIiwiaGFzaGluZ19hbGdvcml0aG0iOiJyYWJiaXRfcGFzc3dvcmRfaGFzaGluZ19zaGEyNTYiLCJ0YWdzIjoiYWRtaW5pc3RyYXRvciJ9XSwKCiAgICAgInZob3N0cyI6W3sibmFtZSI6Ii8ifV0sCiAgICAgInBlcm1pc3Npb25zIjpbeyJ1c2VyIjoiZ3Vlc3QiLCJ2aG9zdCI6Ii8iLCJjb25maWd1cmUiOiIuKiIsIndyaXRlIjoiLioiLCJyZWFkIjoiLioifV0sCiAicGFyYW1ldGVycyI6W10sCiAgICAgImdsb2JhbF9wYXJhbWV0ZXJzIjpbeyJuYW1lIjoiY2x1c3Rlcl9uYW1lIiwidmFsdWUiOiJyYWJiaXRAbG9jYWxob3N0In1dLAogInBvbGljaWVzIjpbXSwKICAgICAicXVldWVzIjpbeyJuYW1lIjoiZmlsZXMiLCJ2aG9zdCI6Ii8iLCJkdXJhYmxlIjp0cnVlLCJhdXRvX2RlbGV0ZSI6ZmFsc2UsImFyZ3VtZW50cyI6e319LAogeyJuYW1lIjoiYXJjaGl2ZWQiLCJ2aG9zdCI6Ii8iLCJkdXJhYmxlIjp0cnVlLCJhdXRvX2RlbGV0ZSI6ZmFsc2UsImFyZ3VtZW50cyI6e319LAogeyJuYW1lIjoic3RhYmxlSURzIiwidmhvc3QiOiIvIiwiZHVyYWJsZSI6dHJ1ZSwiYXV0b19kZWxldGUiOmZhbHNlLCJhcmd1bWVudHMiOnt9fV0sCiJleGNoYW5nZXMiOlt7Im5hbWUiOiJsZWdhIiwidmhvc3QiOiIvIiwidHlwZSI6InRvcGljIiwiZHVyYWJsZSI6dHJ1ZSwiYXV0b19kZWxldGUiOmZhbHNlLCJpbnRlcm5hbCI6ZmFsc2UsImFyZ3VtZW50cyI6e319LAogICAgICAgICAgICAgIHsibmFtZSI6ImNlZ2EiLCJ2aG9zdCI6Ii8iLCJ0eXBlIjoidG9waWMiLCJkdXJhYmxlIjp0cnVlLCJhdXRvX2RlbGV0ZSI6ZmFsc2UsImludGVybmFsIjpmYWxzZSwiYXJndW1lbnRzIjp7fX1dLAogImJpbmRpbmdzIjpbeyJzb3VyY2UiOiJsZWdhIiwgInZob3N0IjoiLyIsICJkZXN0aW5hdGlvbiI6ImFyY2hpdmVkIiwgICJkZXN0aW5hdGlvbl90eXBlIjoicXVldWUiLCAicm91dGluZ19rZXkiOiJhcmNoaXZlZCIsICJhcmd1bWVudHMiOnt9fV0KfQ==" | base64 -d > ega-charts/localega/config/defs.json
-        - cp ega-charts/cega/conf/dummy.key ega-charts/localega/config/dummy.sec
+        # generate secrets
+        - git clone https://github.com/NBISweden/LocalEGA-deploy-init
+        - cd LocalEGA-deploy-init
+        - pip3 install . && cd ..
```

Comment:
Here I suggest two possible options for change:
1)
`pushd LocalEGA-deploy-init`
  `pip install .`
   `popd`
2)
`directly `pip install LocalEGA-deploy-init/`